wazimap_ng/utils.py

```diff
@@ -615,3 +618,24 @@ def clean_columns(file):
     new_columns = df.columns.str.lower().str.strip()
     file.seek(0)
     return old_columns, new_columns
+
+
+def error_handler(request, **kwargs):
+
+    if request.path.startswith('/api/v1'):
```

Comment:
Since this fix is to structure all error messages in json, I'll remove the if statement and render all 500 error responses in json.

Reply:
no, the issue was that all API requests should respond with JSON. 
Not all requests. Otherwise the Admin will show JSON errors and the DRF preview potentially as well.

That's why we need the check for content type here.

Reply:
Fixed. During test , I had to assign content_type to underlying request object so function could be tested without errors.